vocabularies/trs.ttl

```diff
@@ -0,0 +1,178 @@
+# baseURI: http://linked.data.gov.au/def/trs
+# imports: http://www.w3.org/2004/02/skos/core
+# imports: http://www.w3.org/2006/time
+
+@prefix trs: <http://linked.data.gov.au/def/trs#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix time: <http://www.w3.org/2006/time#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/trs>
+  a owl:Ontology ;
+  owl:imports <http://www.w3.org/2004/02/skos/core> ;
+  owl:imports <http://www.w3.org/2006/time> ;
+.
+trs:conceptScheme a skos:ConceptScheme ;
+    skos:prefLabel "Temporal Reference Systems"@en ;
```

Comment:
System (singular), or Systems (plural)? 
Title of vocab is System (singular).

Reply:
@nicholascar as the original submitter of this vocabulary into the GSQ, can you please comment on the feedback from Luke and/or correct the file where necessary?

Reply:
The Concept Scheme should be plural: it contains a number of Temporal Reference System instance. See the Concept Scheme label for [Data Access Rights](https://github.com/geological-survey-of-queensland/vocabularies/blob/master/vocabularies/data-access-rights.ttl) which is also plural.

Reply:
All concept schemes have multiple concepts or they are redundant. In several of our vocabularies we refer to the name as singular. i.e. the selected term from the vocabulary is the Earth Science Data Category, the Sampling Method, the Queensland Resource Permit. Using the GeoSciML as a reference the vocabularies are all singular. http://resource.geosciml.org/def/voc/
It does raise the issue that one way or another these should be consistent throughout all our vocabularies.